Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5838,6 +5838,20 @@ SecureContextChecksEnabled:
     WebCore:
       default: true
 
+SelectShowPickerEnabled:
+  type: bool
+  status: testable
+  category: dom
+  humanReadableName: "Select showPicker() method"
+  humanReadableDescription: "Enable showPicker() method on select"
```

Comment:
Nit: "Select" -> "<select>".

Reply:
Sorry can you clarify what you mean by this?

Reply:
Do you mean "humanReadableName: "showPicker() method""? That would be confusing because it doesn't also control the input.showPicker() method?

Reply:
I mistyped sorry, I meant to say "Select" -> "<select>".

Reply:
Ah that makes sense, GitHub converting that always catches me out too.